### ISSUE TITLE
Facebook invalid_credentials

### ISSUE THREAD

**Issue by stupied4ever** [mkdynamic/omniauth-facebook#276]
Hey guys,
Today suddenly we started to get this error:
auth/failure?message=invalid_credentials&strategy=facebook

Do you have any clue? Looking on Twitter I discovered that more people are having the same issue. There are some FB change?

**Comment by mdoyle13** [comment]
I'm working on this issue as well. It appears FB has changed their API versions 2.3 and below possibly.

**Comment by nandayadav** [comment]
same issue we are having, started today. FB was removing 2.2 today.

**Comment by mdoyle13** [comment]
It's weird because in my initializer i specify
```:client_options => {  
       :authorize_url => "https://www.facebook.com/v2.3/dialog/oauth"
   }```

**Comment by nandayadav** [comment]
updating to latest version of this gem, seems to fix it.

**Comment by stupied4ever** [comment]
@nandayadav seems good, I worked for me

**Comment by kyle-rader** [comment]
@nandayadav 2nd upgrading from to the latest omni-auth that uses graph v2.6 by default solved our problem.

**Comment by stupied4ever** [comment]
I just did a small research and on issue  #245 it was solved. This issue was released under version 4.0.0 on 2016-07-26. 

So, in the end of the day, it's our fault (because we are using the old version), so I will close this issue, and please if somebody has any other problem we can reopen.
